### PR TITLE
lesson-4.1

### PR DESCRIPTION
**Pull request by zemiarsi**

### CODE REVIEW

File: lesson_1.1/src/components/TodoContainer.js [hunk 1]
```diff
@@ -72,4 +70,7 @@ function TodoContainer(props) {
           </div>
   );
 }
+TodoContainer.prototype = {
+tableName:PropTypes.string
```
Comment (kaytijackson97) [inline lesson_1.1/src/components/TodoContainer.js:74]: indentation

File: lesson_1.1/src/components/TodoList.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import TodoListItem from "./TodoListItem";
 import style from './TodoListItem.module.css'
```
Comment (kaytijackson97) [inline lesson_1.1/src/components/TodoList.js:3]: please add semi colon to end of import

File: lesson_1.1/src/components/TodoListItem.js [hunk 3]
```diff
@@ -10,7 +10,6 @@ function TodoListItem({ todo, onRemoveTodo }) {
     <div>
       <li className={style.ListItem} >
         <input type='checkbox' fontSize='medium' className={style.CheckBox}/> 
-       {/* <Checkbox className={style.Checkbox}/>  */}
       <span className={ todo.fields.completed===true? style.CompletedListItem:''}>
```
Comment (kaytijackson97) [inline lesson_1.1/src/components/TodoListItem.js:13]: indentation for lines 13 to 15

**Review by kaytijackson97 — COMMENTED** [review]
Nicely done removing that unused code unprompted! Code is coming together really nicely, still just a couple indentation things. More nit picking is you should remember to add semi colons at the end of import statements and blocks of code. Even though it doesn't affect the functionality of the code, it is still best practice to continue including them

**Comment by zemiarsi** [comment]
Okay I will fixing that too!! Thanks!!

**Pull request merged**